### PR TITLE
Terminology change: rename co-owners to secondary owners

### PR DESCRIPTION
**Pull request by oleg-nenashev**
The idea is to pick low-hanging fruits like classes for Pipeline integration and APIs.
Generally it's not realistic to change everything (enum values, etc.)

Changes:
* "owner" => "primary owner"
* "co-Owner" => "secondary owner"

TODO:
- [x] Base classes - change API to secondaryOwners where it is possible
- [x] Ownership management UIs
- [x] Pipeline documentation and samples
- [x] Ownership variables injection - variable names and documentation
- [x] Role strategy macros
- POSTPONED: Injected environment variables

Change examples:

screen shot 2016-09-15 at 03 41 26
screen shot 2016-09-15 at 03 41 54
screen shot 2016-09-15 at 03 42 07

### CODE REVIEW

**Comment by KostyaSha** [comment]
@oleg-nenashev maybe cast @abayer for grammar-nazi?

**Comment by oleg-nenashev** [comment]
@KostyaSha @abayer 
<code>ABayerCastException: The reviewer is too busy on this week</code>

**Comment by oleg-nenashev** [comment]
I'll handle FreeStyle Build wrappers in a follow-up PR since it requires serious refactoring

File: src/main/java/com/synopsys/arc/jenkins/plugins/ownership/jobs/OwnershipJobFilter.java [hunk 1]
```diff
@@ -78,6 +92,7 @@ public OwnershipJobFilter(String ownerName, boolean acceptCoowners) {
         this.acceptsCoowners = acceptCoowners;
     }
 
+    //TODO: Bug, filtering of abstract projects only
```
Comment (oleg-nenashev) [inline src/main/java/com/synopsys/arc/jenkins/plugins/ownership/jobs/OwnershipJobFilter.java]: Fixed elsewhere

**Comment by KostyaSha** [comment]
👍

**Comment by oleg-nenashev** [comment]
Merging it since it passes manual tests.
The change does not increase technical debt since "primary/secondary owner" terminology is standard at least. The plugin still requires documentation copy-editing

**Pull request merged**